lib/kafo/kafo_configure.rb

```diff
@@ -166,6 +166,13 @@ def logger
     def run(*args)
       started_at = Time.now
       logger.debug("Running installer with args #{args.inspect}")
+      if config.app[:verbose]
+        log_output_message = <<~MESSAGE.strip
+          Running installer with log based terminal output at level #{config.app[:verbose_log_level].upcase}.
+          Use -l to set the terminal output log level to ERROR, WARN, NOTICE, INFO, or DEBUG.
```

Comment:
```suggestion
          Use -l to set the terminal output log level to ERROR, WARN, NOTICE, INFO, or DEBUG.
          Or use --progress-bar to display a simple progress bar instead of log based terminal output.
```

@ehelms @ekohl thoughts on including this as well?

Reply:
Given the progress bar is broken, and we were tyring to make a more permanent move to the log based output. I would opt for keeping that part out.

Reply:
That sounds good. Based on conversation today on this topic, we may want to discuss dropping the progress bar entirely.